test/tests/kernels/fv_adapt/transient-adapt.i

```diff
@@ -0,0 +1,116 @@
+[Mesh]
+  type = GeneratedMesh
```

Comment:
It is better to use a `mesh generator?`   Other input files in this PR use `mesh generator`.

I know this is very minor. It is up to you.

Reply:
changed to use the generator